theano/tensor/nnet/tests/test_nnet.py

```diff
@@ -659,7 +661,7 @@ def test_softmax_grad_optimizations(self):
         fgraph = gof.FunctionGraph(
                 [x, one_of_n],
                 [g_x])
-        self.assertTrue(hasattr(fgraph.outputs[0].tag, 'trace'))
```

Comment:
According to the check below, I think we should check for `crossentropy_softmax_1hot_with_bias_dx` and `softmax_op`.